src/app/page.tsx

```diff
@@ -1,39 +1,29 @@
 'use client'
-import { CastComponent } from '@/components/cast-component';
-import { ProfileComponent } from '@/components/profile-component';
-import { FIDPFP, exampleCast, warpcastURLCast, warpcastURLCastURL, warpcastURLPFP, warpcastURLProfile } from '@/constants';
 
+import { CastComponent } from '@/components/cast-component'
+import LatestCasts from '@/components/latest-casts'
+import { ProfileComponent } from '@/components/profile-component'
+import {
+  FIDPFP,
+  exampleCast,
+  warpcastURLCast,
+  warpcastURLCastURL,
+  warpcastURLPFP,
+  warpcastURLProfile,
+} from '@/constants'
 
 export default function Home() {
-
   return (
-    <div className="flex flex-col items-center justify-center space-y-4 p-2">
-      <div className='flex flex-col items-center space-y-4 p-5 m-5'>
-      <div className='flex flex-row space-x-2'>
-<div className='flex flex-col space-y-2 items-center justify-center '>
-
-  <p className='text-center'>Example hash</p>
-        <CastComponent cast={exampleCast} />
-        </div>
-        <div className='flex flex-col space-y-2 w-full items-center justify-center '>
-       <p className='text-center'>Example Warpcast cast url</p>
-       <CastComponent cast={warpcastURLCast} warpcastUrl={warpcastURLCastURL} />
-        </div>
-        </div>
-        <div className='flex flex-row space-x-2'>
-       <div className='flex flex-col space-y-2 w-full items-center justify-center '>
-       <p className='text-center'>Example FID</p>
-        <ProfileComponent pfp={FIDPFP} url={"3"} />
-        </div>
-        <div className='flex flex-col space-y-2 w-full items-center justify-center '>
-       <p className='text-center'>Example Warpcast profile url</p>
-       <ProfileComponent pfp={warpcastURLPFP} url={warpcastURLProfile} />
-       
+    <div className="flex flex-col items-center space-y-4 mt-10 w-full">
+        <div className="flex flex-col gap-2 md:flex-row md:gap-0 justify-center md:justify-between w-[100%]">
+          <CastComponent cast={exampleCast} headerText="Example hash" />
+          <CastComponent cast={warpcastURLCast} warpcastUrl={warpcastURLCastURL} headerText="Example Warpcast cast url" />
         </div>
+        <div className="flex flex-col gap-2 md:flex-row md:gap-0 justify-center md:justify-between w-[100%]">
+          <ProfileComponent pfp={FIDPFP} url="3" headerText="Example FID" />
+          <ProfileComponent pfp={warpcastURLPFP} url={warpcastURLProfile} headerText="Example Warpcast profile url" />
         </div>
-      
-      </div>
+        <LatestCasts />
```

Comment:
can you make it so the latest casts are in the center of the screen, seems like they aren't rn 
<img width="924" alt="Screenshot 2024-06-05 at 10 32 14 PM" src="https://github.com/neynarxyz/explorer/assets/17347501/3a25a22b-d56a-4fad-8bb4-f32c3a41610b">